ghost/mailgun-client/lib/mailgun-client.js

```diff
@@ -172,12 +172,19 @@ module.exports = class MailgunClient {
         const providerId = event?.message?.headers['message-id'];
 
         return {
+            id: event.id,
             type: event.event,
             severity: event.severity,
             recipientEmail: event.recipient,
             emailId: event['user-variables'] && event['user-variables']['email-id'],
             providerId: providerId,
-            timestamp: new Date(event.timestamp * 1000)
+            timestamp: new Date(event.timestamp * 1000),
+
+            error: event['delivery-status'] && event['delivery-status'] ? {
+                code: event['delivery-status'].code,
+                message: event['delivery-status'].message || event['delivery-status'].description,
```

Comment:
While normally it shouldn't happen, but think we should restrict the final value here for message to `2000` chars as that's the max allowed length in our DB. Will allow us to capture the error in case mailgun returns a message/description which is longer instead of failing due to DB limit, wdyt ? 
Same thing for `enhanced_code` as well.

Reply:
Cool, I'll add a length restriction